### ISSUE TITLE
Merge the two sign-in pages

### ISSUE THREAD

**Issue by zsusag** [CSC322-Grinnell/boardbank#57]
Right now we have two separate sign in pages for `Users` and `Organizations`. This could be a accessibility issue and should be merged into a single `Sign In` page.
